Makefile

```diff
@@ -83,4 +83,87 @@ docs-serve:
 	poetry run mkdocs serve
 
 docs-build:
-	poetry run mkdocs build
\ No newline at end of file
+	poetry run mkdocs build
+
+# Docker
+AWS_REGION=eu-west-2
```

Comment:
Should we move this into the `.env.example` directory for central configuration? (Under an AWS section)

Reply:
it is already in the .env

Reply:
this is tricky as we already have some AWS variables in the .env... and they overlap with the minio variables. This will need some thought so I have created a TD ticket to address it https://technologyprogramme.atlassian.net/jira/software/projects/REDBOX/boards/409?selectedIssue=REDBOX-156